docs/extend/plugins.md

```diff
@@ -1,87 +1,272 @@
 <!--[metadata]>
 +++
-title = "Extending Engine with plugins"
-description = "How to add additional functionality to Docker with plugins extensions"
-keywords = ["Examples, Usage, plugins, docker, documentation, user guide"]
+aliases = [
+"/engine/extend/"
+]
+title = "New Plugin System"
```

Comment:
See my earlier comment on the *history* (although perhaps it's partly due to the double move, once from @vieux's commit, and once from @sfsmithcha's); seems to be a GitHub issue though: if I checkout this PR, and check `git blame` for `docs/extend/legacy_plugins.md`:

```
git blame docs/extend/legacy_plugins.md

c4d45b6a docs/extend/plugins.md               (David Calavera       2015-07-15 11:25:56 -0700  1) <!--[metadata]>
c4d45b6a docs/extend/plugins.md               (David Calavera       2015-07-15 11:25:56 -0700  2) +++
e310d070 docs/extend/plugins.md               (Mary Anthony         2016-01-23 20:36:40 -0800  3) title = "Extending Engine with plugins"
c4d45b6a docs/extend/plugins.md               (David Calavera       2015-07-15 11:25:56 -0700  4) description = "How to add additional functionality to Docker with plugins extensions"
c4d45b6a docs/extend/plugins.md               (David Calavera       2015-07-15 11:25:56 -0700  5) keywords = ["Examples, Usage, plugins, docker, documentation, user guide"]
c4d45b6a docs/extend/plugins.md               (David Calavera       2015-07-15 11:25:56 -0700  6) [menu.main]
e310d070 docs/extend/plugins.md               (Mary Anthony         2016-01-23 20:36:40 -0800  7) parent = "engine_extend"
a7a70433 docs/extend/legacy_plugins.md        (Charles Smith        2016-08-16 11:53:36 -0700  8) weight=3
c4d45b6a docs/extend/plugins.md               (David Calavera       2015-07-15 11:25:56 -0700  9) +++
c4d45b6a docs/extend/plugins.md               (David Calavera       2015-07-15 11:25:56 -0700 10) <![end-metadata]-->
c4d45b6a docs/extend/plugins.md               (David Calavera       2015-07-15 11:25:56 -0700 11)
a7a70433 docs/extend/legacy_plugins.md        (Charles Smith        2016-08-16 11:53:36 -0700 12) # Understand legacy Docker Engine plugins
```

The history looks to be there